contrib/python/scripts/ascii2netcdf.py

```diff
@@ -0,0 +1,166 @@
+'''
+This function creates netcdf files from an input ascii file 
+formatted following the ASPECT convention for structured ascii data files.
+The generated netcdf file can be used for visualization in Paraview or 
+as an input file in ASPECT.
+
+see https://unidata.github.io/netcdf4-python/ for more information
+
+'''
+
+import netCDF4 as nc
+import numpy as np
+import sys
+
+def main():
+
+    msg = "Usage            : ascii2netcdf.py input_ascii_filename  [dimensions, coordinate_system, output_format] \n \n" + \
+          "Optional arguments: \n" + \
+          "dimensions       : '2' or '3', the dimensions of the data in the ascii file \n" +  \
+          "coordinate_system: 'spherical' or 'cartesian' \n" +  \
```

Comment:
What if the ASCII file is something different like a P-T table? Would a third option that just preserves the columns as is possible?

Reply:
Something like in the latest commit ? I added another function to take care of such files. I also forgot about the ascii depth profile files.